src/gluonts/dataset/artificial/_base.py

```diff
@@ -84,7 +84,7 @@ def __init__(
         self,
         num_timeseries: int = 10,
         num_steps: int = 30,
-        time_granularity: str = "1H",
+        freq: str = "1H",
```

Comment:
This needs to be kept backwards compatible in the json files for now. Use

```
freq: str = pydantic.Schema(..., alias='time_granularity', schema: '1H')
```

in order to allow use with existing `metadata.json` files.

Reply:
FYI you can use `# type: ignore` to silence mypy for this line.